contributed_definitions/NXimage_set_em_kikuchi.nxdl.xml

```diff
@@ -0,0 +1,322 @@
+<?xml version="1.0" ?>
+<?xml-stylesheet type="text/xsl" href="nxdlformat.xsl"?>
+<definition category="base" extends="NXobject" name="NXimage_set_em_kikuchi" type="group" xmlns="http://definition.nexusformat.org/nxdl/3.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.w3.org/2001/XMLSchema-instance">
+    <symbols>
+        <symbol name="n_p">
+            <doc>
+                 Number of scan points, one pattern per scan point.
+            </doc>
+        </symbol>
+        <symbol name="n_y">
+            <doc>
+                 Number of pixel per Kikuchi pattern in the slow direction
+            </doc>
+        </symbol>
+        <symbol name="n_x">
+            <doc>
+                 Number of pixel per Kikuchi pattern in the fast direction
+            </doc>
+        </symbol>
+    </symbols>
+    <doc>
+         Electron backscatter diffraction (EBSD) Kikuchi pattern
+         
+         The container can also store data related to a post-processing of these
+         Kikuchi pattern, which is the backbone of orientation microscopy
+         especially in materials science and materials engineering.
+         
+         Based on a fuse of the [h5ebsd](https://doi.org/10.1186/2193-9772-3-4)
```

Comment:
markdown-style URL convert to reST (as before)

Reply:
fixed